decksite/controllers/api.py

```diff
@@ -143,6 +157,41 @@ def decks_api() -> Response:
     resp.set_cookie('page_size', str(page_size))
     return resp
 
+@APP.api.route('/scraper/')
+class Scraper(Resource):
+    @APP.api.marshal_with(MULTIPLE_DECKS)
+    def get(self) -> Dict[str, Any]:
+        """
+        Grab a slice of finished sorted decks last updated after a certain point.
+        Input:
+            {
+                'sortBy': <str>,
+                'sortOrder': <'ASC'|'DESC'>,
+                'page': <int>,
+                'pageSize': <int>,
+                'lastTimestamp': <int>,
+                'seasonId': <int|'all'>
+            }
+        Output:
+            {
+                'page': <int>,
+                'objects': [<deck>],
+                'total: <int>
+            }
+        """
+        season = seasons.season_id(str(request.args.get('seasonId')), None)
+        order_by = 'GREATEST(cache.active_date, q.changed_date) DESC'
```

Comment:
Do you need to order by this? It might be expensive and it seems unnecessary if you're already providing `since`?

Reply:
that might explain the bad speed, yea this probably makes sense

Reply:
If I don't add `order_by` at all, it will still order in a way consistent between requests right? so the pagination actually would work

Reply:
Good point. Maybe try `ORDER BY d.id` and compare the performance to this?